src/utils/commandRunner.ts

```diff
@@ -59,12 +59,18 @@ export class CommandRunner {
         `${executable} ${args}`,
         mountPaths
       );
-      runEnv = undefined;
```

Comment:
Passing all local environment variables within EE can be problematic. The logic can be updated to pass any environment variable starting with `ANSIBLE_`.
We are this on ansible-navigator with EE. Refer https://github.com/ansible/ansible-navigator/blob/main/src/ansible_navigator/runner/base.py#L204

Reply:
This seems like a great idea.
+1 for this logic !